src/pages/use-cases/kube-proxy.jsx

```diff
@@ -1,3 +1,4 @@
+import PageTitle from 'components/pages/use-cases/page-title';
```

Comment:
LGTM! Only one thing that you forgot to remove the PageTitle, that's why the build failed.

Reply:
yep, just updated it